teams/docs/membership.json

```diff
@@ -0,0 +1,59 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "lilin90",
+        "yikeke"
```

Comment:
@lilin90 @queenypingcap @TomShawn @qiancai

### A bit background

We're undergoing sig merging phase and the role "sig leader" is a bit different from "team maintainers".

To be precise, a maintainer, based on the rights of a committer, is able to promote or evict(emeritus member for reference) a reviewer, a committer or a maintainer by majority decision.

And they are responsible for
* Decide the overall direction of the team.
* Speaking on behalf of the team.
* Maintaining guidelines of the team.
* Nominating new team members.

Team maintainers come from sig leader, but we hope have more maintainers who are responsible to grow the community, not one or two "leaders".

### What you need to do

Since @yikeke left PingCAP and was inactive for a while, I'd prefer move her as an emeritus maintainer.

And we need more maintainers to take the responsibility of growing the community. May @qiancai, @TomShawn & @queenypingcap good candidate.

What do you think?

Reply:
@tisonkun Thanks for the detailed explanation and the proposal.    

I would like to recommend adding @TomShawn and me to team maintainers. @lilin90, would you please evaluate the request? Thanks!

Reply:
> @tisonkun Thanks for the detailed explanation and the proposal.
> 
> I would like to recommend adding @TomShawn and me to team maintainers. @lilin90, would you please evaluate the request? Thanks!

I agree with adding @TomShawn and @qiancai to team maintainers.

Reply:
Thanks for your reply! Updating...

Reply:
Updated.